server.py

```diff
@@ -432,6 +432,26 @@ def put(self, region, id):
             tournament.regions = args['regions']
 
         dao.update_tournament(tournament)
+
+    def delete(self, region, id):
+        """ Deletes a pending tournament.
+            If a (non-pending) tournament with this id exists, tell the user they can't do that.
+            Route restricted to admins for this region. """
+        dao = Dao(region, mongo_client=mongo_client)
+
+        user = get_user_from_access_token(request.headers, dao)
+        if not is_user_admin_for_region(user, region):
+            return 'Permission denied', 403
+
+        pending_tournament = dao.get_pending_tournament_by_id(ObjectId(id))
+        if not pending_tournament:
+            tournament = dao.get_tournament_by_id(ObjectId(id))
+            if tournament:
+                return "Cannot delete a finalized tournament.", 400
+            else:
+                return "No pending tournament found with that id.", 400
+
+        dao.delete_pending_tournament(pending_tournament)
```

Comment:
You should probably return some sort of response here. A JSON {"success": true} or something would be fine.